wp-dependency-installer.php

```diff
@@ -88,25 +109,24 @@ public function load_hooks() {
 		public function run( $plugin_path ) {
 			if ( file_exists( $plugin_path . '/wp-dependencies.json' ) ) {
 				$config = file_get_contents( $plugin_path . '/wp-dependencies.json' );
-				if ( empty( $config ) ||
-					null === ( $config = json_decode( $config, true ) )
-				) {
-					return;
-				}
-				$this->source = basename( $plugin_path );
-				$this->load_hooks();
-				$this->register( $config );
```

Comment:
> `$this->load_hooks();`

Is this the correct position? ( **before** the `$this->register( $config )` function ? )